src/CTA.WebForms/ClassConverters/PageCodeBehindClassConverter.cs

```diff
@@ -50,11 +53,14 @@ public override Task<IEnumerable<FileInformation>> MigrateClassAsync()
             //     // This is so we can use ComponentBase base class
             //     .Append(Constants.BlazorComponentsNamespace);
 
+            var currentClassDeclaration = await DoTagCodeBehindConversions((ClassDeclarationSyntax)_originalDeclarationSyntax);
```

Comment:
always check for null before doing type casting

Reply:
In this case it should be impossible for _originalDeclarationSyntax to be null because we have to use the syntax node to determine that we need to initialize the current class instance, but will add a check regardless for safety